### PR TITLE
Update configure-proxy-internet.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information about what URL's to open in firewall.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5751

File: microsoft-365/security/defender-endpoint/configure-proxy-internet.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ The following downloadable spreadsheet lists the services and their associated U
 |
 
 If a proxy or firewall has HTTPS scanning (SSL inspection) enabled, exclude the domains listed in the above table from HTTPS scanning.
+In your firewall, open all the URLs where the geography column is WW. For rows where the geography column is not WW, open the URLs to your specific data location. [You can verify your data location setting.](/microsoft-365/security/defender-endpoint/data-retention-settings)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-proxy-internet.md]:
```suggestion
In your firewall, open all the URLs where the geography column is WW. For rows where the geography column is not WW, open the URLs to your specific data location. To verify your data location setting, see [Verify data storage location and update data retention settings for Microsoft Defender for Endpoint](/microsoft-365/security/defender-endpoint/data-retention-settings).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @msbemba, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**